benchmarks/static_runtime/deep_wide_pt.h

```diff
@@ -31,6 +32,77 @@ struct DeepAndWide : torch::nn::Module {
   torch::Tensor mu_, sigma_, fc_w_, fc_b_;
 };
 
+// Implementation using native functions and pre-allocated tensors.
+// It could be used as a "speed of light" for static runtime.
+struct DeepAndWideFast : torch::nn::Module {
```

Comment:
might want to check in a test against the non-fast version

Reply:
wdym?